### PR TITLE
Add library system name & service point ID fields to Library

### PR DESCRIPTION
**Pull request by humble-ahitofel**
Add two basic fields without any smarter validation (library systems can have very specific id formats).
Note: API v4 has a separate merge request for this later.

### CODE REVIEW

File: templates/entity/Library/edit.html.twig [hunk 1]
```diff
@@ -44,6 +44,15 @@
...
         {{ form_row(form.identificator) }}
+        <div class="row">
+          <div class="col">
+            {{ form_row(form.library_system_name)}}
+          </div>
+          <div class="col">
+            {{ form_row(form.library_system_id)}}
+          </div>
+        </div>
+
```
Comment (ajsalminen) [inline templates/entity/Library/edit.html.twig]:
```suggestion
```

File: src/Module/ApiCache/Serializer/Normalizer/LibraryNormalizer.php [hunk 2]
```diff
@@ -78,13 +78,24 @@ public function normalize($object, $format = null, array $context = [])
             $values['customData'] = $this->extractCustomData($object->getCustomData());
         }
 
+        
+
```
Comment (ajsalminen) [inline src/Module/ApiCache/Serializer/Normalizer/LibraryNormalizer.php]: Just a couple of unnecessary whitespace lines.

```suggestion
```

Comment (ajsalminen) [inline src/Module/ApiCache/Serializer/Normalizer/LibraryNormalizer.php]:
```suggestion
```

File: data/database.alter.sql [hunk 3]
```diff
@@ -2176,3 +2176,10 @@ ALTER TABLE schedules DROP COLUMN staff;
 
 
 ALTER TABLE finna_additions ADD COLUMN custom_data jsonb;
+
+
+ALTER TABLE organisations ADD library_system_name varchar(255);
+COMMENT ON COLUMN organisations.library_system_name IS 'Machine name of the system in use (sierra, axiell, koha)';
+
+ALTER TABLE organisations ADD library_system_id varchar(255);
```
Comment (ajsalminen) [inline data/database.alter.sql]: After our discussion about what this represents my thought here is that maybe it would be better to call this field "library_system_servicepoint_id". Would that be an appropriate name and are there any issues changing it? The benefit would be that it becomes clear at first look that it doesn't mean a library system (instance or system type) specific identifier but something that identifies some more specific part of the library operations like one service point.

The comment on the column could maybe be a little more specific about the purpose too.

Comment (humble-ahitofel) [reply data/database.alter.sql]: Should renaming `library_system_id` to `library_system_servicepoint_id` reflect to everything else, like for example the kirkanta api (query param names, json payload, etc)?

Comment (ajsalminen) [reply data/database.alter.sql]: Yeah I think that would be ideal if it would since it cuts through all those as the same thing as far as I understand, does doing that cause any additional issues?

Comment (humble-ahitofel) [reply data/database.alter.sql]: No, not really. I will implement the changes now.

Comment (humble-ahitofel) [reply data/database.alter.sql]: `library_system_id` has been now renamed. I revised a little the sql commenting.

File: data/database.alter.sql [hunk 4]
```diff
@@ -2176,3 +2176,10 @@ ALTER TABLE schedules DROP COLUMN staff;
 
 
 ALTER TABLE finna_additions ADD COLUMN custom_data jsonb;
+
+
+ALTER TABLE organisations ADD library_system_name varchar(255);
+COMMENT ON COLUMN organisations.library_system_name IS 'Machine name of the library system that is used currently in the orgainsation (sierra, axiell, koha)';
```
Comment (ajsalminen) [inline data/database.alter.sql]:
```suggestion
COMMENT ON COLUMN organisations.library_system_name IS 'Machine name of the library system that is used currently in the organisation (sierra, axiell, koha)';
```

File: data/database.alter.sql [hunk 5]
```diff
@@ -2176,3 +2176,10 @@ ALTER TABLE schedules DROP COLUMN staff;
...
+
+ALTER TABLE organisations ADD library_system_servicepoint_id varchar(255);
+COMMENT ON COLUMN organisations.library_system_servicepoint_id IS 'ID given by the library system to the organsation (Sierra, Axiell, Koha, etc.)';
```
Comment (ajsalminen) [inline data/database.alter.sql]:
```suggestion
COMMENT ON COLUMN organisations.library_system_servicepoint_id IS 'ID given by the library system to the organisation (Sierra, Axiell, Koha, etc.)';
```

**Comment by ajsalminen** [comment]
I made some minor changes to documentation.md but didn't generate the html version Should be good to go after that, can you update it?
